stored_queries/GO/GO_get_terms_from_ws_feature.yaml

```diff
@@ -0,0 +1,44 @@
+# Get the terms from a feature
+
+name: GO_get_terms_from_ws_feature
+params:
+  type: object
+  required: [feature_id, ts]
+  properties:
+    feature_id:
+      type: string
+      title: Workspace versioned object reference
+    limit:
+      type: integer
+      title: Maximum result limit
+      default: 20
+      maximum: 1000
+    offset:
+      type: integer
+      title: Result offset for pagination
+      default: 0
+      maximum: 100000
+    ts:
+      type: integer
+      title: Versioning timestamp
+query_prefix: WITH GO_terms
+query: |
+  LET results=(
+    FOR f in ws_genome_features
+      FILTER f.feature_id == @feature_id
```

Comment:
I think we will want the input & filter to be on `f._key` rather than `f.feature_id`, which will be globally unique. I think there's enough of a risk that feature IDs will not be globally unique, so we want the genome ID prepended to make sure we don't get ambiguous results.

Also, we may need to have some auth control on this query using ws_ids, since the genomes that these features are attached to may be private. But I'm not sure about this. We can ask in standup soon